enclave-runner/src/usercalls/mod.rs

```diff
@@ -545,6 +552,43 @@ fn trap_attached_debugger(tcs: usize) {
     }
 }
 
+fn connect_stream_dflt(
```

Comment:
Move this implementation into `fn connect_stream` below. This way you can handle the `OutputBuffer`s the same way for both the default and custom cases.